doc/manual/manual.tex

```diff
@@ -2268,10 +2268,87 @@ \subsection{Particles}
+
+\item Compile \& Install (advanced users and developers with the need to reconfigure underlying libraries or running massively parallel models): The most advanced option is to compile and install \aspect{} from source. This allows maximal control over the underlying libraries like Trilinos and deal.II, as well as easy modifications to \aspect{} by recompiling a modified source directory. Our installation instructions cover most Linux and MacOS operating systems, but incompatibilities on individual systems can always occur and make the installation more cumbersome. If you are planning to run massively parallel computations on a compute cluster this is likely your only option. Since clusters usually have a very individual setup, it is always a good advice to ask IT support staff for help when installing \aspect{}, to avoid hard to reproduce setup problems, and performance penalties.
+\end{enumerate}
+
+\subsection{Docker Container}
+
+\subsubsection{Installing Docker and downloading the \aspect{} image}
+
+Docker is a lightweight virtualization software that allows to ship applications with all their dependencies in a simple way. It is outside of the scope of this manual to explain all possible applications of Docker, and we refer to the introduction (\url{https://www.docker.com/what-docker}) and installation and quickstart guides (\url{https://www.docker.com/products/docker}) on the Docker website for more detailed descriptions of how to set up and use the docker engine. More importantly Docker provides a marketplace for exchanging prepared docker images (called Docker Hub). After setting up the docker engine downloading a precompiled \aspect{} image from Docker Hub is as simple as typing in a terminal:
+
+\begin{verbatim}
+docker pull gassmoeller/aspect
+\end{verbatim}
+
+\subsubsection{Running \aspect{} models}
+Although it is possible to use the downloaded \aspect{} docker image in a number of different ways, we recommend the following workflow:
+
+\begin{enumerate}
+\item Create your \aspect{} input file in a folder of your choice (possibly also containing any input model data that is required by your model).
+\item Run the docker image and mount the prepared directory as a volume into the docker container. The code below assumes you have navigated into the prepared directory. When you have started the container run the aspect model inside the container, and make sure the model output is saved in your prepared directory.
+\begin{verbatim}
+docker run -it -v "$(pwd):/home/dealii/aspect/model" \
```

Comment:
I am a bit confused. These points seem not longer as critical as they used to be (at least I did not encounter them), but we can certainly discuss them in the manual. 
On my machine when mounting the directory it appears with uid and permissions for the docker user specified in the dockerfile ('dealii') inside of the container. From the outside all files in that directory (newly created or changed) have my (host) user's permissions and uid, so somehow docker now translates these uids and permissions (or is this by accident because dealii and my host user happen to have the same uid?). When I accidentally mount a directory that belongs to root, it also appears as root inside of the container, and the dealii user is not allowed to change those. It is also not possible to run su or sudo from within the container, therefore you can not accidentally break things or change permissions. I do not know about mounting other users directories, but can give it a try tomorrow. I am not sure if this depends on the type of docker storage driver (I use Aufs under ubuntu), but they seem to handle file permissions pretty well nowadays. Of course in a user created container the user could make himself root and do crazy things, but in our container with a prescribed non-root user without sudo-privileges it seems safe.
Addition: Just tested windows and had some issues getting write-access to my mounted volumes (read access works fine). I will give it a closer look tomorrow.

About 2.: We can discuss those points, but in fact everything you do in that container is kept (I do not know if that was changed recently as well). When you kill the container or it exits normally its data is kept in the virtual file system of that container (not the image), and you can restart this container by finding its name or id via `docker ps -a`, and all created data will still be there. In fact this means if you often start new containers via `docker run image` (without the -rm option), and create data within the container you need to clean up every now and then (`docker container prune` removes all finished containers). From the docker documentation: `By default a container’s file system persists even after the container exits.` I agree that it is bad practice for us to create data outside of the mounted volume, because it might invisibly fill up user's hard drives, but it is not lost, and one might for example also create a workflow that only mounts a volume read-only, and then manually copy out the data. This was in fact my first workflow description, which is why the `docker container prune` command further down was left over (it needs to be removed now).